apps/storybook/src/SurfaceVis.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { SurfaceVisProps } from '@h5web/lib';
+import { getDomain, SurfaceVis } from '@h5web/lib';
+import { createArrayFromView, getMockDataArray } from '@h5web/shared';
+import { OrbitControls } from '@react-three/drei';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+
+import FillHeight from './decorators/FillHeight';
+
+const dataArray = createArrayFromView(
+  getMockDataArray('/nD_datasets/fourD').pick(0, 0, null, null)
+);
+const domain = getDomain(dataArray.data);
+
+const Template: Story<SurfaceVisProps> = (args) => (
+  <SurfaceVis {...args}>
+    <OrbitControls />
+  </SurfaceVis>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  dataArray,
+  domain,
+};
+
+export default {
+  title: 'Experimental/SurfaceVis',
```

Comment:
The new `SurfaceVis` can be tested in the Storybook and is deemed as **Experimental**